testsuite/forge.py

```diff
@@ -532,7 +542,7 @@ def get_axiom_link_for_test_runner_logs(
         ['k8s.labels.app.kubernetes.io/name'] = "forge" and ['k8s.namespace'] == "{forge_namespace}"
         """
 
-    logs_url = f"https://app.axiom.co/aptoslabs-hghf/explorer?initForm={urlquote( json.dumps({ 'apl': apl_query, 'queryOptions': apply_axiom_time_filter(time_filter), }) )}"
+    logs_url = f"https://app.axiom.co/aptoslabs-hghf/explorer?initForm={urlquote(json.dumps({'apl': apl_query, 'queryOptions': apply_axiom_time_filter(time_filter), }))}"
```

Comment:
Do we still use Axiom ?

Reply:
I am keeping the door open for now, in case we need to re-evaluate.